packages/ringcentral-integration/modules/DetailedPresence/index.js

```diff
@@ -59,7 +59,7 @@ export default class DetailedPresence extends Presence {
   _subscriptionHandler = (message) => {
     if (presenceRegExp.test(message.event) && message.body) {
       if (message.body.sequence) {
-        if (message.body.sequence <= this._lastSequence) {
+        if (message.body.sequence < this._lastSequence) {
```

Comment:
I think when we remove '=', we need to consider if it has any impact for call secenario.
I think QAs need to do call scenario tests as many as possible.